### ISSUE TITLE
Read movies from a file instead of a folder 

### ISSUE THREAD

**Issue by eklipse18** [utkarshbajaj/Find-IMDB-Ratings#15]
I think this would be a better approach because making a separate folder just to scrape movies is just unecessary trouble. Insteasd, the user could make a file containing the movies he wants to get information of.

**Comment by Vthechamp22** [comment]
can I work on this?

**Comment by bastienlogeot** [comment]
Hello 
is it still possible to work to this project?
i could help you with this feature

**Comment by utkarshbajaj** [comment]
Yeah sure, why not. Make sure to implement this as an option (if-else) rather than only the file and no folder. Have fun coding!
